gamehype/forms.py

```diff
@@ -28,3 +36,21 @@ def validate_email(self, email):
         user = User.query.filter_by(email=email.data).first()
         if user is not None:
             raise ValidationError('Please use a different email address.')
+
+
+class AddGameForm(FlaskForm):
+    game_name = StringField(
+        'Game Title',
+        [validators.required(), validators.length(min=2, max=128)]
```

Comment:
```suggestion
        validators=[DataRequired(), validators.length(min=2, max=128)]
```

Reply:
I was just going off of another guide that had multiple validators listed out this way. If this still works locally, I'll keep it for consistency.